cmd/rcd/rcd.go

```diff
@@ -54,6 +55,20 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 			if err := checkRelease(rcflags.Opt.WebGUIUpdate); err != nil {
 				log.Fatalf("Error while fetching the latest release of rclone-webui-react %v", err)
 			}
+			if rcflags.Opt.NoAuth{
+				rcflags.Opt.NoAuth = false
+				log.Println("Cannot run web-gui without authentication, using default auth")
```

Comment:
Can you make these logs with the rclone logging system?

So use `log.Noticef(nil, "Cannot run...")`

The log levels are explained here - there is a corresponding function for each one

https://rclone.org/docs/#log-level-level

Reply:
Maybe I am missing something, but I'm unable to find ```log.Noticef()``` or ```fs.Noticef()```.